.changeset/seven-laws-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'extract-react-types': patch
+---
+
+Allow proper conversion of keys in TypeScript types that are quoted.
```

Comment:
Maybe, 

> Adds support for string type keys in TypeScript

Reply:
That sounds a lot better! I was having trouble describing this haha